### ISSUE TITLE
NPR: Increased circulating docosatetraenoate level

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#4386]
**NPR:** Increased circulating docosatetraenoate level

**Parent**
HP_0008167 Very Long Chain Fatty Acid Accumulation

**For LOINC code:**
35182-5 Docosatetraenoate (C22:4w6) [Moles/volume] in Serum or Plasma

**Biocurator ID for loinc2hpo:** UCDenver:tjcallahan

**Comment by pnrobinson** [comment]
There is a synonym Adrenic-acid, which is easier -- let's use it as the preferred label
https://pubchem.ncbi.nlm.nih.gov/compound/Adrenic-acid

**Comment by pnrobinson** [comment]
@callahantiff also note that the accumulation term is wrong -- this belongs under http://cthpo02lp.jax.org:8080/app/browse/term/HP:0004359

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pubmed/28456640

**Comment by pnrobinson** [comment]
added as HP_0020158
